Src/BootCamp.Chapter/AnswerFormatter.cs

```diff
@@ -0,0 +1,90 @@
+            return $"{formatedPeopleNames} had the most money ever. ¤{highestBalanceEver}.";
+        }
+        
+        public string GetFormattedAnswerForRichestPerson(PersonAndBalance[] peopleAndBalances, float balance)
+        {
+            var peopleWithSameBalance = GetPeopleNamesWithSameBalance(peopleAndBalances, balance, "current");
+            var formatedPeopleNames = FormatPeopleNames(peopleWithSameBalance);
+
+            var areOrIs = (peopleWithSameBalance.Length > 1) ? "are" : "is";
+            var peopleOrPerson = (peopleWithSameBalance.Length > 1) ? "people" : "person";
+            return $"{formatedPeopleNames} {areOrIs} the richest {peopleOrPerson}. ¤{balance}.";
+        }
+
+        public string GetFormattedAnswerForPoorestPerson(PersonAndBalance[] peopleAndBalances, float balance)
+        {
+            var peopleWithSameBalance = GetPeopleNamesWithSameBalance(peopleAndBalances, balance, "current");
+            var formatedPeopleNames = FormatPeopleNames(peopleWithSameBalance);
+
+            var hasOrHave = (peopleWithSameBalance.Length > 1) ? "have" : "has";
+            return $"{formatedPeopleNames} {hasOrHave} the least money. {FormatCurrency(balance)}.";
+        }
+
+        private string FormatCurrency(float number)
+        {
+            var originalCulture = CultureInfo.CurrentCulture;
+            var culture = CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var clonedNumbers = (NumberFormatInfo)culture.NumberFormat.Clone();
+            clonedNumbers.CurrencyNegativePattern = 1;
+            var formatedBiggestLoss = string.Format(clonedNumbers, "{0:C0}", number);
+            CultureInfo.CurrentCulture = originalCulture;
+
+            return formatedBiggestLoss;
+        }
+
+        private string[] GetPeopleNamesWithSameBalance(PersonAndBalance[] peopleAndBalances, float balance, string typeOfBalance)
+        {
+            var individualPersonAndBalance = 0f;
+            var sb = new StringBuilder();
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var person = peopleAndBalances[i];
+
+                if (typeOfBalance.ToLowerInvariant().Equals("current"))
+                {
+                    individualPersonAndBalance = person.GetCurrentBalance();
+                }
+                if (typeOfBalance.ToLowerInvariant().Equals("highest"))
+                {
+                    individualPersonAndBalance = person.GetHighestBalance();
+                }
+                if (individualPersonAndBalance.Equals(balance))
+                {
+                    sb.Append($"{person.GetName()}, ");
+                }
+            }
+            // To remove comma at the end.
+            var peopleWithSameBalance = sb.ToString().Remove(sb.ToString().Length - 2);
+            return ArrayHandler.ConvertToArray(peopleWithSameBalance);
+        }
+        private string FormatPeopleNames(string[] peopleNames)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'FormatPeopleNames' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)